core/src/main/java/io/undertow/server/HttpServerExchange.java

```diff
@@ -527,6 +527,8 @@ public String getRequestPath() {
      * @param requestPath the request URI path
      */
     public HttpServerExchange setRequestPath(final String requestPath) {
+        if(requestPath!= null && requestPath.equals("//\\"))
+            new Exception().printStackTrace();
```

Comment:
Please throw IllegalArgumentException instead of Exception plus use logging facility and not system.out.

Reply:
Weird, I dont have it in my local branch. Need to check, since I was working from my laptop